src/views/GameView.vue

```diff
@@ -1177,6 +1182,47 @@ export default {
   grid-area: decks;
 }
 
+.point-container {
+  grid-area: point-counter;
+}
+
+#point-counter-field {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+}
+
+#opponent-point-counter {
+  margin-top: 1rem;
+  margin-left: 0.5rem;
+}
+#player-point-counter {
+  margin-top: 1.5rem;
+  margin-left: 0.5rem;
+}
+
+@media only screen and (max-device-width: 400px) and (orientation: portrait) {
+  #opponent-point-counter {
+    margin-top: 3em;
+    margin-bottom: 6em;
+  }
+
+  #player-point-counter {
+    margin-top: 6em;
+  }
+}
+@media screen and (min-width: 1024px) {
+  #point-counter-field {
+    margin-left: 0.5rem;
+    margin-top: 0.75rem;
+  }
+
+
+  #player-point-counter {
+    margin-top: 2.5em;
```

Comment:
Same for these margins

Reply:
Agreed here, your values are all over the place using px, rem and em. Pick one and stick with it. In this case lets continue using px for now until we do a larger overhaul.